repos/rust-lang/mentorship-team.toml

```diff
@@ -0,0 +1,8 @@
+org = "rust-lang"
+name = "mentorship-team"
+description = "Home of the Rust mentorship team"
+homepage = "https://mentorship-team.rust-lang.org/"
```

Comment:
Is this automatically generated by the website for each team? :eyes:

Reply:
Well, specifically what that does is just add the link to the sidebar like you can see in:

https://github.com/rust-lang/lang-team/

Then the team's repo includes the automation to use GH pages for the website.  Probably someone then does a manual step to add the DNS entry and whatnot for this.

Reply:
I've gone ahead and pulled that out of the commit.  It's fine to just have the charter and whatnot in the `README.md` for now.  If you do want to create a full website like lang and some other teams do, and add the automation for this, we could add this back.  Or I can put it back in now if you think you will actually want this.

Reply:
I was wondering whether this is automated, because e.g. `https://infra-team.rust-lang.org/` is 404. I don't think we need to have a homepage link in there for now.

Reply:
Actually, now that you mention infra, I'm reminded that many teams including them link to their pages on `rust-lang.org`, e.g. <https://www.rust-lang.org/governance/teams/infra>.  That seems like the right default.  I've updated it to that.

Reply:
Yeah, that's indeed a good default. Thanks!